webpack/scenes/ContentViews/Details/Versions/Compare/__tests__/CVVersionCompare.test.js

```diff
@@ -368,31 +399,31 @@ test('Can select two versions with no content and click compare button', async (
   const scope = nockInstance
     .get(cvVersions)
     .query(true)
-    .reply(200, emptyStateCVVersionsData);
+    .reply(200, cvVersionsData);
```

Comment:
Why are we doing this? The 'Can select two versions with no content and click compare button' test name doesn't reflect what the test is doing anymore?

Reply:
Oh..nevermind..I see version 1 and 2 in fixture are empty so this should be good.